composer.json

```diff
@@ -25,13 +25,15 @@
         "influxdb/influxdb-php": "^1.15",
         "posthog/posthog-php": "^3.0",
         "influxdata/influxdb-client-php": "^3.4",
-        "guzzlehttp/guzzle": "^7.5"
+        "guzzlehttp/guzzle": "^7.5",
+        "promphp/prometheus_client_php": "^2.3"
```

Comment:
Why is this a dev dependency and in the readme you are instructing devs to manually add the requirement - im confused!

Reply:
That's the way they do it for all other clients like influx, posthog etc. They are all listed as suggestions, so the client installs only whats needed

Reply:
I get the suggestions, but this is in the dev dependency, and not a suggestion